docker/build_test_ntf.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+configure() {
+    cd /workspace
+    PATH="$PATH:$(realpath srcs/bde-tools/bin)"
+    export PATH
+    eval "$(bbs_build_env -u opt_64_cpp17)"
+}
+
+build_ntf() {    
+    cd /workspace/ntf-core
+    sed -i s/CMakeLists.txt//g ./configure
+    ./configure --prefix /opt/bb
+    make -j8
+    make test | tee test.log
```

Comment:
Failed `make test` will not cause this script to fail, so I have to save its log and check it later